fern/docs.yml

```diff
@@ -74,36 +88,22 @@ navigation:
               - page: Parameter Names
                 icon: fa-regular fa-input-text
                 path: ./pages/api-definition/openapi/extensions/parameter-names.mdx
-                slug: parameter-names
-              - page: Audiences
-                icon: fa-duotone fa-users
-                path: ./pages/api-definition/openapi/extensions/audiences.mdx
-                slug: audiences              
+                slug: parameter-names          
```

Comment:
fyi that my plan is to eventually get rid of the broad `Extension` category and just have docs for each relevant concept (e.g. Audiences, Streaming, Examples, etc.)